platform/security/idp/security-idp-plugin/security-idp-plugin-subjectconfirmation/pom.xml

```diff
@@ -69,6 +69,10 @@
               platform-util,
               platform-util-unavailableurls
             </Embed-Dependency>
+            <Import-Package>
+              javax.annotation;version="[1.1,2)",
```

Comment:
I am not clear _why_ this needed to be added to this module directive.

Reply:
I looked into this a bit more. It's coming from the dependency tree: ddf-security-common -> cxf-rt-frontend-jaxrs -> javax.annotations 1.3. I don't think we're starting up a bundle exporting javax.annotations 1.3 ¯\_(ツ)_/¯ Not sure if this will impact much. Was this failing at build or runtime? If at runtime,  we could start up the javax.annotations 1.3 bundle as well. Not sure if it's worth it if nothing is broken though.

Reply:
Runtime. It may not be necessary any longer, as I'm installing the annotations jar in the `lib/ext` directory.